### ISSUE TITLE
Different languages not showing properly

### ISSUE THREAD

**Issue by SID177** [rtMediaWP/rtMedia#1567]
We need to add `<?xml encoding="utf-8" ?>` while using `DOMDocument` class.

Without adding utf-8 encoding.
![Screenshot 2020-03-03 at 5 04 21 PM](https://user-images.githubusercontent.com/23107546/75771997-1f894980-5d71-11ea-880e-3045e8f82581.png)

With utf-8 encoding
![Screenshot 2020-03-03 at 5 05 46 PM](https://user-images.githubusercontent.com/23107546/75772082-447dbc80-5d71-11ea-97c9-37524867f0bd.png)

**Comment by pooja-muchandikar** [comment]
@SID177 

Tested with `pre-release` branch and the text in the activity page looks fine now. Tested with german and french language. 

```
Test Environment
WordPress: v5.4-RC-1
rtMedia: v4.6.0 pre-release branch
BuddyPress: v5.1.2 Nouveau and Legacy Template
Theme: 2020 and 2019
```

**Comment by SID177** [comment]
Demo texts
```
Meine Familie ist sehr groß. Ich habe zwei Schwestern, einen Bruder, drei Tanten, einen Onkel und sechs Cousins. Meine große Schwester hat lange blonde Haare und heißt Laura und eine kleine Schwester heißt Miranda und ist dunkelhaarig. Mein Bruder heißt Fred und trägt eine Brille. Ich verstehe mich gut mit meiner kleinen Schwester und meinem Bruder. Mit meiner großen Schwester streite ich mich oft um den Computer. Mein Vater arbeitet zwar viel, aber am Wochenende hilft er uns immer bei den Hausaufgaben. Meine Mutter backt gerne Torten. Ihre Schokotorten mag ich besonders gerne. In den Ferien besuchen wir häufig meine Großeltern, da sie leider so weit entfernt wohnen. Meine anderen Großeltern, die Eltern meiner Mutter wohnen eine Straße weiter. Das finde ich schön, da wir uns oft sehen können. Außerdem haben sie eine süße Perserkatze, mit der ich immer spiele. Wenn uns meine Cousins besuchen kommen, unternehmen wir meist etwas Besonderes. Letztes Wochenende waren wir alle zusammen im Zoo. Das war lustig, da mein Cousin Ben Angst vor Schlangen hatte. Ich mag meine Familie!Meine Familie ist sehr groß. Ich habe zwei Schwestern, einen Bruder, drei Tanten, einen Onkel und sechs Cousins. Meine große Schwester hat lange blonde Haare und heißt Laura und eine kleine Schwester heißt Miranda und ist dunkelhaarig. Mein Bruder heißt Fred und trägt eine Brille. Ich verstehe mich gut mit meiner kleinen Schwester und meinem Bruder. Mit meiner großen Schwester streite ich mich oft um den Computer. Mein Vater arbeitet zwar viel, aber am Wochenende hilft er uns immer bei den Hausaufgaben. Meine Mutter backt gerne Torten. Ihre Schokotorten mag ich besonders gerne. In den Ferien besuchen wir häufig meine Großeltern, da sie leider so weit entfernt wohnen. Meine anderen Großeltern, die Eltern meiner Mutter wohnen eine Straße weiter. Das finde ich schön, da wir uns oft sehen können. Außerdem haben sie eine süße Perserkatze, mit der ich immer spiele. Wenn uns meine Cousins besuchen kommen, unternehmen wir meist etwas Besonderes. Letztes Wochenende waren wir alle zusammen im Zoo. Das war lustig, da mein Cousin Ben Angst vor Schlangen hatte. Ich mag meine Familie!
```

```
Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն Հայերեն
```

```
ქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართულიქართული
```

Paste this text inside activity, refresh the page and check whether it's showing properly or not.

**Comment by kjnanda** [comment]
Okay, so this look good now.

**Comment by kjnanda** [comment]
This is working as expected and tested with `dev` branch. ✅

**Comment by kjnanda** [comment]
Closing this issue as this has been fixed and release in - V4.6.1
